go.mod

```diff
@@ -45,6 +47,7 @@ require (
 	github.com/swaggo/http-swagger v1.2.6
 	github.com/swaggo/swag v1.8.3
 	github.com/syndtr/goleveldb v1.0.1-0.20190318030020-c3a204f8e965
+	github.com/tikv/pd/client v0.0.0-20230710071111-47d408b547b9
```

Comment:
I suggest not adding the client here.

Reply:
I adjusted the code to remove this dependency. PTAL.